packages/grafana-ui/src/components/Logs/LogDetails.tsx

```diff
@@ -102,72 +126,89 @@ class UnThemedLogDetails extends PureComponent<Props> {
   };
 
   render() {
-    const { row, theme, onClickFilterOutLabel, onClickFilterLabel, getRows } = this.props;
+    const {
+      row,
+      theme,
+      onClickFilterOutLabel,
+      onClickFilterLabel,
+      getRows,
+      showDuplicates,
+      className,
+      onMouseEnter,
+      onMouseLeave,
+    } = this.props;
     const style = getLogRowStyles(theme, row.logLevel);
+    const styles = getStyles(theme);
     const labels = row.labels ? row.labels : {};
     const labelsAvailable = Object.keys(labels).length > 0;
     const fields = this.getAllFields(row);
     const parsedFieldsAvailable = fields && fields.length > 0;
 
     return (
-      <div className={style.logDetailsContainer}>
-        <table className={style.logDetailsTable}>
-          <tbody>
-            {labelsAvailable && (
-              <tr>
-                <td colSpan={5} className={style.logDetailsHeading} aria-label="Log Labels">
-                  Log Labels:
-                </td>
-              </tr>
-            )}
-            {Object.keys(labels).map(key => {
-              const value = labels[key];
-              return (
-                <LogDetailsRow
-                  key={`${key}=${value}`}
-                  parsedKey={key}
-                  parsedValue={value}
-                  isLabel={true}
-                  getStats={() => calculateLogsLabelStats(getRows(), key)}
-                  onClickFilterOutLabel={onClickFilterOutLabel}
-                  onClickFilterLabel={onClickFilterLabel}
-                />
-              );
-            })}
+      <tr className={className} onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
```

Comment:
I have replaced the  `cursor: pointer ` with `cursor: default` when hovering over log details.  I don't think that we should hide details when clicked on log details table, as this could interfere with other clickable logDetailsTable elements and could be potentially confusing. Very good catch Andrej!